lib/sentry/client.ex

```diff
@@ -91,6 +92,24 @@ defmodule Sentry.Client do
     end
   end
 
+  @spec send_client_report(ClientReport.t()) ::
+          {:ok, client_report_id :: String.t()} | {:error, ClientError.t()}
+  def send_client_report(%ClientReport{} = client_report) do
+    # We don't pass options because this is internal. But should we use default client or users?
+    client = Config.client()
```

Comment:
No, always use the client configured by the user. They might not even have the default client as a dependency.